tests/DCPS/BuiltInTopicTest/DataReaderListener.cpp

```diff
@@ -32,7 +31,8 @@ void DataReaderListenerImpl::on_data_available(DDS::DataReader_ptr reader)
   try {
     ::Messenger::MessageDataReader_var message_dr = ::Messenger::MessageDataReader::_narrow(reader);
     if (CORBA::is_nil (message_dr.in ())) {
-      cerr << "DataReaderListener: read: _narrow failed." << endl;
+      ACE_ERROR((LM_ERROR, ACE_TEXT(
```

Comment:
With this change, will the run_test.pl or prst_repo_run_test.pl stdout/stderr be useful?  Maybe the -ORBLogFile args need to go away?  In general, different tests have different approaches to this.  We can discuss offline too.

Reply:
The reason I made the test this way what that when I began working on this test in earnest I was thinking it will inherently would print error messages that needed to be ignored. This was the case in the past but that doesn't seem to happen now.